cogs/mod.py

```diff
@@ -45,6 +47,37 @@ def __init__(self, bot):
             await ctx.send(default.actionmessage("kicked"))
         except Exception as e:
             await ctx.send(e)
+            
+    @commands.has_permissions(administrator=True)
+    @commands.command()
+    async def say(self, ctx, *, msg=None):
+        if ctx.message.attachments:
+            file = BytesIO()
+            att = ctx.message.attachments[0]
+            await att.save(file)
+            file.seek(0)
+            await ctx.send(msg, file=discord.File(file, filename=att.filename))
+        else:
+            await ctx.send(msg)
+        await ctx.message.delete()
+    
+    @commands.command()

```

Comment:
I won't add softban considering it's a copy of ban/unban command while also being easy to make yourself if you really desire to have such command.